docs/quickstarts/packet.md

```diff
@@ -5,262 +5,307 @@
+application to verify the cluster behaves as expected.
 
-### Step 1: Install lokoctl
+## Requirements
 
-lokoctl is a command-line interface for Lokomotive.
+* A Packet account with a project created.
+* A Packet project ID.
+* A Packet
+  [user level API key](https://www.packet.com/developers/docs/API/getting-started/)
+  with access to the relevant project.
+* An AWS account.
+* An AWS
+  [access key ID and secret](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)
+  of a user with
+  [permissions](https://github.com/kinvolk/lokomotive/blob/master/docs/concepts/dns.md#aws-route-53)
+  to edit Route 53 records.
+* An AWS Route 53 zone (can be a subdomain).
+* An SSH key pair for accessing the cluster nodes.
+* Terraform `v0.12.x`
+  [installed](https://learn.hashicorp.com/terraform/getting-started/install.html#install-terraform).
+* The `ct` Terraform provider `v0.5.0`
+  [installed](https://github.com/poseidon/terraform-provider-ct#install).
+* `kubectl` [installed](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+>NOTE: The `kubectl` version used to interact with a Kubernetes cluster needs to be compatible with
+>the version of the Kubernetes control plane. Ideally you should install a `kubectl` binary whose
+>version is identical to the Kubernetes control plane included with a Lokomotive release. However,
+>some degree of version "skew" is tolerated - see the Kubernetes
+>[version skew policy](https://kubernetes.io/docs/setup/release/version-skew-policy/) document for
+>more information. You can determine the version of the Kubernetes control plane included with a
+>Lokomotive release by looking at the [release notes][releases].
 
-To install `lokoctl`, follow the instructions in the [lokoctl installation](../installer/lokoctl.md)
-guide.
+## Steps
 
-### Step 2: Set up a working directory
+### Step 1: Install lokoctl
 
-It's better to start fresh in a new working directory, as the state of the cluster is stored in this
-directory.
+`lokoctl` is the command-line interface for managing Lokomotive clusters.
 
-This also makes the cleanup task easier.
+Download the `lokoctl` binary for your platform from the latest release on the
+[releases] page:
 
 ```console
-mkdir -p lokomotive-infra/mycluster
-cd lokomotive-infra/mycluster
+wget https://github.com/kinvolk/lokomotive/releases/download/v0.1.0/lokoctl_0.1.0_linux_amd64.tar.gz
 ```
 
-### Step 3: Set up credentials from environment variables
-
-#### Packet
-
-* Log in to your Packet account and obtain the Project ID from the `Project Settings` tab.
-* Obtain an API key from the `User Settings` menu.
-* Set the environment variable `PACKET_AUTH_TOKEN` with the API key.
+Extract the binary and copy it to a place under your `$PATH`:
 
 ```console
-export PACKET_AUTH_TOKEN=<PACKET_API_KEY>
+tar zxvf lokoctl_0.1.0_linux_amd64.tar.gz
+sudo cp lokoctl_0.1.0_linux_amd64/lokoctl /usr/local/bin
+rm -rf lokoctl_0.1.0_linux_amd64*
```

Comment:
Sorry to change my mind, but I just realize we have assets for non-linux too, and this example hardcodes Linux. While I think it is not bad, maybe we can just remove them and assume everyone knows how to run `tar xf`?

Other alternatives are like:
```
export platform=linux
wget <something with $platform>
```

But, for the sake of brevity just say "download ..." as it says now  and just add "include it to your path" (without instructions) might be fine?

No strong preference, if using only linux as an example (and wasn't an oversight, I didn't know we release for Mac), I'm fine regarding this

Reply:
I really wanted to include copy-paste commands which just work. The problem is that it's tricky to programmatically point to the latest release when pre-releases are used. I've resorted to telling the user to find the latest release themselves and included the `wget` and `tar` commands as *examples*, not as something to copy-paste as is. I can look into it a bit further to see if I can find a better way.

Reply:
I've managed to programmatically get the latest version using `curl` + `jq` trickery. I'm not sure it's better though since the commands are complex now and have more dependencies.